pkg/kubelet/cm/cpumanager/policy_options_test.go

```diff
@@ -70,18 +70,6 @@ func TestPolicyOptionsAvailable(t *testing.T) {
 			featureGateEnable: true,
 			expectedAvailable: false,
 		},
-		{
-			option:            FullPCPUsOnlyOption,
```

Comment:
I don't know this feature - are these testcases GONE or just covered elsewhere?

Reply:
Uhm, good point. Since we're graduating an option this specific test was implicitely covered elsewhere, but changing the test to be more explicit won't hurt

Reply:
handled explicitely in lines 134-148 below